packages/perseus/src/components/__stories__/input-with-examples.stories.tsx

```diff
@@ -1,53 +1,48 @@
-import * as React from "react";
+import {actions} from "@storybook/addon-actions";
 
 import InputWithExamples from "../input-with-examples";
 
-type StoryArgs = Record<any, any>;
+import type {Meta, StoryObj} from "@storybook/react";
 
-type Story = {
-    title: string;
+const meta: Meta = {
+    title: "Perseus/Components/Input with Examples",
+    component: InputWithExamples,
+    args: {
+        examples: [],
+        id: "",
+        onChange: actions("onChange"),
+        value: "",
+    },
+    argTypes: {
+        onChange: {
+            table: {disable: true},
+        },
```

Comment:
Why do you want to remove `onChange` from the table?

Reply:
It didn't seem useful because you can't edit it. Do you find in WonderBlocks that it's useful to see callback props in the Controls table?

Reply:
Personally, I think it's helpful even if I can't change it in the controls table, because it still lets me know what props are available. I use the controls table for documentation in addition to the manual control.

Reply:
Ok, that's convincing. I've updated all the stories I changed to just disable the controls instead of hiding them.